app/src/oculusvr/cpp/DeviceDelegateOculusVR.cpp

```diff
@@ -40,10 +40,6 @@
 
 namespace crow {
 
-const vrb::Vector kAverageHeight(0.0f, 1.7f, 0.0f);
-// Height used to match Oculus default in WebVR
-const vrb::Vector kAverageOculusHeight(0.0f, 1.65f, 0.0f);
```

Comment:
You can leave this file untouched, we don't plan to maintain the deprecated OculusVR backend. We're just keeping it there for a while for reference, but will eventually remove it.